perceval/backends/core/confluence.py

```diff
@@ -379,9 +386,11 @@ class ConfluenceClient(HttpClient):
     VEXPAND = ['body.storage', 'history', 'version']
     VHISTORICAL = 'historical'
 
-    def __init__(self, base_url, archive=None, from_archive=False, ssl_verify=True, spaces=None):
+    def __init__(self, base_url, archive=None, from_archive=False, ssl_verify=True,
+                 spaces=None, max_contents=MAX_CONTENTS):
```

Comment:
Neither `spaces` nor `max_contents` are in the documentation of this class. Please add them.